docs/integrations/sources/shopify-migrations.md

```diff
@@ -1,5 +1,17 @@
 # Shopify Migration Guide
 
+## Upgrading to 2.6.0
+This version completely deprecates the following streams:
+- `Products Graph QL`
+- `Customer Saved Search` 
+
+To replace the old `Products Graph QL` use the `Products` instead.
+There is no effective replacement for the `Customer Saved Search`. This stream would be re-implemented using `GraphQL BULK` soon.
+
+### Action items required for 2.6.0
+Please, disable the affected streams to continue syncing the data. Use the stream equivalent instead.
```

Comment:
I think the users do not have to do anything as [the default value for raise_exception_on_missing_stream is false](https://github.com/airbytehq/airbyte-python-cdk/blob/0e7802a4c485355d8d5abaa5f23b48b682dc4eb7/airbyte_cdk/sources/abstract_source.py#L224-L226). I would prefer to have to user have as little to worry as possible and hence would remove this section. Does that make sense?

Reply:
To be clear, they won't be removed from the catalog but the source is resilient to that so it doesn't matter. It is still probably cleaner if the user refresh their schema but I don't think this is necessary

Reply:
It's removed. Thanks.